dotcom-rendering/src/amp/components/Ad.tsx

```diff
@@ -111,28 +104,34 @@ interface CommercialConfig {
 	useAmazon: boolean;
 }
 
-export interface AdProps {
-	isSticky?: boolean;
+interface BaseAdProps {
 	edition: Edition;
 	section: string;
 	contentType: string;
-	config: CommercialConfig;
 	commercialProperties: CommercialProperties;
 }
 
-export interface RegionalAdProps extends AdProps {
-	adRegion: AdRegion;
+export interface AdProps extends BaseAdProps {
+	isSticky?: boolean;
+	rtcConfig: string;
 }
 
-export const RegionalAd = ({
+export interface RegionalAdProps extends BaseAdProps {
+	config: CommercialConfig;
+}
+
+export interface StickyAdProps extends BaseAdProps {
+	config: CommercialConfig;
+}
```

Comment:
This is a nitpick but can we move this definition to the StickyAd file since that's where it's used?

Reply:
Good call!